index.html

```diff
@@ -147,10 +147,15 @@ <h2>Scope</h2>
         </p>
         <ul>
           <li>Addressing errata, ambiguities, and interoperability problems found in previous versions of the Verifiable Credentials Data Model</li>
-          <li>Data models for credentials, presentations, and proofs</li>
-          <li>Registries for the data models</li>
+          <li>A data model for Verifiable Credentials, inclusive of:</li>
+            <ul>
+                <li>credentials and verifiable credentials</li>
+                <li>presentations and verifiable presentations</li>
+                <li>requests for or submissions of verifiable credentials or verifiable presentations</li>
```

Comment:
What is the intension of this line? it's already stated that we are defining a data model, will data model for "requests or submissions" be different from a "general" data model?

Reply:
As stated above (https://github.com/w3c/vc-wg-charter/pull/60#discussion_r799477016), we are trying to be more precise about what is in scope for the VCWG. Specifically, the line above covers "presentation requests" -- that is, "how does one request a verifiable presentation/credential?" The VCWG 1.0 covered how you express a presentation and verifiable presentation, but it did not cover how a verifier requests one (though, those mechanisms have existed for at least 6 years, we were waiting for the market to mature to the point where we could have multiple inputs in order to standardize them).